src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyDefinition.cs

```diff
@@ -141,17 +141,37 @@ private JToken GetResource()
             var policyDefinitionObject = new PolicyDefinition
             {
                 Name = this.Name,
-                Properties = new PolicyDefinitionProperties
-                {
-                    Description = this.Description ?? null,
-                    DisplayName = this.DisplayName ?? null,
-                    PolicyRule = this.GetObjectFromParameter(this.Policy, nameof(this.Policy)),
-                    Metadata = this.Metadata == null ? null : this.GetObjectFromParameter(this.Metadata, nameof(this.Metadata)),
-                    Parameters = this.Parameter == null ? null : this.GetObjectFromParameter(this.Parameter, nameof(this.Parameter)),
-                    Mode = string.IsNullOrEmpty(this.Mode) ? PolicyDefinitionMode.All : this.Mode
-                }
+                Properties = new PolicyDefinitionProperties()
             };
 
+            var policyObject = this.GetObjectFromParameter(this.Policy, nameof(this.Policy));
+            if (policyObject["policyRule"] != null)
```

Comment:
Should be "properties.policyRule" (and properties.metadata, etc...) right)

Reply:
I chose a policy format without the property bag (you can see the sample file in this PR). This matches what's displayed by the VSCode extension and the portal (when creating a new policy). It also seems more consistent with the format used for the policyRule-flavored input (the policy rule isn't inside ```"policyRule": {}``` or ```"properties": { "policyRule": {} }```). Since this input comes from users, I figured supporting copy-and-paste from VSCode and the portal was a good idea. Let me know if you think I should change it.